packages/core/package.json

```diff
@@ -31,19 +31,19 @@
   },
   "scripts": {
     "build": "tsc --build",
-    "lint": "npm run prettier -- --check && npm run eslint && npm run api-extractor",
-    "lint:fix": "npm run prettier -- --write && npm run eslint -- --fix",
+    "lint": "pnpm prettier --check && pnpm eslint && pnpm api-extractor",
+    "lint:fix": "pnpm prettier --write && pnpm eslint --fix",
     "eslint": "eslint \"src/**/*.{ts,tsx}\" \"test/**/*.{ts,tsx}\" \"test-integrations/**/*.{ts,tsx}\"",
     "prettier": "prettier \"**/*.{js,ts,md,json}\"",
-    "preapi-extractor": "npm run build",
+    "preapi-extractor": "pnpm build",
     "api-extractor": "api-extractor run --local --verbose",
     "test": "mocha --recursive \"test/**/*.ts\"",
-    "test:debug": "DEBUG='ignition:*' npm run test",
+    "test:debug": "DEBUG='ignition:*' pnpm test",
     "test:build": "tsc --project ./test/",
     "test:coverage": "nyc mocha --recursive \"test/**/*.ts\" \"test-integrations/**/*.ts\"",
     "test:integrations": "mocha --recursive \"test-integrations/**/*.ts\"",
     "clean": "rimraf .nyc_output coverage dist tsconfig.tsbuildinfo",
-    "prepack": "npm run build"
```

Comment:
I think to run pre/post scripts in pnpm you need to enable: `enable-pre-post-scripts=true` in the `.npmrc` file:

https://pnpm.io/cli/run#enable-pre-post-scripts

Reply:
`enable-pre-post-scripts` is enabled by default as of version 9